### DISCUSSION TITLE
What is your JavaScript Algorithms and Data Structures workflow/application?

### DISCUSSION THREAD

**Discussion by jakegros** [USU-ITLS-x265-23-summer/class-repo#110] [Q&A]
Has anyone found a helpful workflow when working through the JS Algorithms and Data Structures?
Which ones have you found most helpful?  I would really appreciate any pointers on any applications you have in mind.

**Comment by kassyoveson** [comment]
There are many programs and books that can help with workflow and data structures. I have found that learning the basics helps a lot. Practice working with JS Algorithms. It takes time, if it was easy, then everyone would do it. Don't be afraid to ask for help. We are all struggling with JS. Okay, maybe not all, but I know I am. Take breaks when you need to. I take my dog on a walk to clear my mind and let off some steam. Chocolate and coke also help. :)
